CHANGELOG.md

```diff
@@ -77,6 +77,8 @@ Main (unreleased)
 
 ### Enhancements
 
+- Flow: Add retries with backoff logic to Phlare write component. (@ctovena)
```

Comment:
Can you merge/rebase on top of main and move this to the unreleased enhancements section?